### PR TITLE
Add issue template forms

### LINKED ISSUE #18: Upgrade our issue templates to use GitHub issue forms ✍️
# Introduction

GitHub has recently rolled out a [public beta for their issue forms](https://github.blog/changelog/2021-06-23-issues-forms-beta-for-public-repositories/) feature. This would allow you to create interactive issue templates and validate them 🤯.

Appwrite currently uses the older [issue template format](https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/about-issue-and-pull-request-templates). Your task is to create **GitHub issue forms** for this repository. Please use [Appwrite's issue templates](https://github.com/appwrite/appwrite/tree/master/.github/ISSUE_TEMPLATE) as a reference for this PR.

### Tasks summary:

- [ ] Fork & clone this repository
- [ ] Prepare **bug report** issue form in `.github/ISSUE_TEMPLATE/bug.yaml`
- [ ] Prepare **documentation** issue form in `.github/ISSUE_TEMPLATE/documentation.yaml`
- [ ] Prepare **feature request** issue form in `.github/ISSUE_TEMPLATE/feature.yaml`
- [ ] Push changes to **master** and test issue forms on your fork
- [ ] Submit pull request

If you need any help, reach out to us on our [Discord server](https://discord.gg/GSeTUeA).

Are you ready to work on this issue? 🤔 Let us know, and we will assign it to you 😊

**Happy Appwriting!**

### PR DESCRIPTION
**Pull request by Sven65**
Closes #18

- [x] Fork & clone this repository
- [x] Prepare bug report issue form in .github/ISSUE_TEMPLATE/bug.yaml
- [x] Prepare documentation issue form in .github/ISSUE_TEMPLATE/documentation.yaml
- [x] Prepare feature request issue form in .github/ISSUE_TEMPLATE/feature.yaml
- [x] Push changes to master and test issue forms on your fork
- [x] Submit pull request

### CODE REVIEW

**Comment by lohanidamodar** [comment]
@Sven65  Great work. One more change before we can approve. We have made few changes in the language and also the format of Code of conduct. Please check https://github.com/appwrite/appwrite template and update accordingly.
Thank you.

**Comment by Sven65** [comment]
Thanks @lohanidamodar! I've added the missing code of conduct checks!

File: .github/ISSUE_TEMPLATE/bug.yaml
Comment (lohanidamodar) [inline .github/ISSUE_TEMPLATE/bug.yaml:37]: We can remove this.

File: .github/ISSUE_TEMPLATE/bug.yaml
Comment (lohanidamodar) [inline .github/ISSUE_TEMPLATE/bug.yaml:49]:
```suggestion
    id: php-runtimes-version
```

File: .github/ISSUE_TEMPLATE/bug.yaml
Comment (lohanidamodar) [inline .github/ISSUE_TEMPLATE/bug.yaml:51]:
```suggestion
      label: "🎲 PHP Runtimes version"
```

File: .github/ISSUE_TEMPLATE/bug.yaml
Comment (lohanidamodar) [inline .github/ISSUE_TEMPLATE/bug.yaml:52]:
```suggestion
      description: "What version of PHP Runtimes are you running?"
```

**Review by lohanidamodar — CHANGES_REQUESTED** [review]
@Sven65 great job. Small changes to make it consistent 👍🏻

**Comment by christyjacob4** [comment]
@Sven65  Let us know if you're still working on this

**Comment by Sven65** [comment]
I thought I had submitted the fixes, but apparently not, will do.

**Review by lohanidamodar — APPROVED** [review]

**Comment by christyjacob4** [comment]
THANK YOU! All changes merged 🥳

Please reach out to me on our [Discord server](https://appwrite.io/discord) if you would like to claim your Appwrite swags! As a way of saying thank you, we would also love to invite you to join the Appwrite organization on GitHub. Please share your GitHub username with us on Discord.  

You can accept the invite by visiting https://github.com/orgs/appwrite/invitation. By joining our team, you will officially be an Appwrite maintainer on GitHub.

You can change your membership [visibility settings](https://help.github.com/en/github/setting-up-and-managing-your-github-user-account/publicizing-or-hiding-organization-membership), so your new Appwrite team membership badge will show up on your personal GitHub profile.

Please feel free to look for more PRs you might be interested in helping with on our long list of [Hacktoberfest friendly issues](https://github.com/search?q=org%3Aappwrite+org%3Autopia-php+is%3Aissue+label%3Ahacktoberfest+created%3A%3E2021-01-01+is%3Aopen&type=issues) and help make Appwrite better :)

**Pull request merged**
